vars.yml

```diff
@@ -1,13 +1,35 @@
 ---
-name_prefix: ansible-tower-test
-vsi_image: ibm-redhat-8-3-minimal-amd64-3
-vsi_profile: bx2-4x16
-#ssh_public_key: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC2xCcL+l/hHGI0s8qfx3sm9JV4tnTSmYzimr7t4KmYEi+RUODyKtvU+LkyedYGhdWXIMgx5yajafAoz8oacMX+SctNa9RAXz31pBC53rztKYnlvly806BXg6s0fTei1gtVt5SGhApviWGVjyQ89CkON1rM41mBqiS9015ND3D2kYBVu3g1+0t/6Tp4bSZpIdFAAf95jDtkqTMo04+c73JqQA/pyXguU7NpAmIC7YMyRXwow8Q0UTiBkcgk7Z9/HCxXz7elfx48nTko3HHgwTublODLdnOiG1BgWESLtLP5H+NvU22bUJ32nnN1WQ9RhrMHxcb8H4TsKLM257fT0I5O8an8TQ/ulhX3+pO1SFpdv6NVAgmhokIjP08VHPYknn8V5GO0q5wx11/8Gk4eT8LKrjhLJThongXxbAWKBz9pVpsnumvQH8F1i9MduJxbq980ZjRAjyNvz2RuN5RFWixCTPidpqtNi1p6uoBSyNvTZ5TzpnJhwtdxAKpL9qX6PZ8= root@localhost-localdomain'
-  #total_ipv4_address_count: 256
-ssh_public_key: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDlkniCZbiIhOv9RkoSE73Yg4sjRtuiuXWKHBCvhQ9BxZV33rMEGCUwYs2XT8QWbuJxBg+aXspZ1Of00lNV1S2/Ukg4n6Os7szI6mqJQgFR/oqLckGbHML76CdvwQ1iVoSFdE5bieQdfbK+LmIRVQAzO93090jWtWaJ8MkxzgarvXpPcwidVjB/PDsIq3NG/6boH/xUhxRH/Qt+4ShVH0/w4YTLGAvp56yeN+1XWoGy/D0V7JQX9y82ca3n9S+DsYAJBcUp/k7rjqtEsNBDYpNk7nvjmt9jiXCVCBu6EajcINOa/uEsWsyX/iyxE6XENsBP+hPK7/HMEqvMJw2cfJakCHCLTRpYZSOS4fursT53ZXt+UWko8S2xIqt/kkH4HSvrfHejlUWGILlRhEKX6SfMQaLLWaqz8QT9LIhEY/gRjk4wekAYWSQziaMeEcUsLVITihLKKG8qw6+ig5TOlw6LAhBJxCsKPkWLul7wAP5j6mI9/M73nX8JXX3O/drcNR0= pok@localhost.localdomain'
-image_id: 'r022-939bbfe2-d823-4c01-98a3-a6f8193731d8'
+# Ansible Tower Installation
 #installation_tarball: https://releases.ansible.com/ansible-tower/setup-bundle/ansible-tower-setup-bundle-latest.tar.gz
 installation_tarball: http://perf54.perf.lab.eng.bos.redhat.com/pub/ansible-automation-platform-setup-bundle-2.0.0-1-early-access.tar.gz
-ic_instance: 3
-list_of_ports: [22, 80, 443, 5432, 9121, 9256, 9117, 9393, 9392, 9187, 9100]
 inventory_template: inventory_cluster_4.j2
+
+# Tower and the Database must be in the same Region and in the same VPC
+ibmcloud_vpc_name_prefix: 'ansible-tower-test'
+ibmcloud_vpc_region: 'jp-tok'    # Tokyo
+ibmcloud_vpc_image_name: 'ibm-redhat-8-3-minimal-amd64-3'
+ibmcloud_vsi_image_id: 'r022-939bbfe2-d823-4c01-98a3-a6f8193731d8'
+ibmcloud_vsi_profile: 'bx2-4x16'
+ibmcloud_vsi_count: 2 
+install_tower: True
+
+# Ansible Tower and Tower database must be in the same 
+ibmcloud_vsi_db_profile: 'bx2-4x16'
+ibmcloud_vsi_db_count: 1    # Required database nodes should be always 1
+
+
+# Isolated Nodes must be in the different region compared to Controller
+ibmcloud_vpc_iso_name_prefix: 'ansible-tower-iso-test'
+ibmcloud_vpc_iso_region: 'eu-gb'    # London   
+ibmcloud_vsi_iso_image_name: 'ibm-redhat-8-3-minimal-amd64-3'
+ibmcloud_vsi_iso_image_id: 'r018-179f9b5c-0e4d-4614-8fbb-07e8ec55a493'
+ibmcloud_vsi_iso_profile: 'bx2-4x16'
+ibmcloud_vsi_iso_instance_count: 0

```

Comment:
Should we make this as a default of 1? It will be little counter intuitive for users to change both install_iso and make this value as 1 if they want to install iso nodes. In any case if install_iso is set to False, even if ibmcloud_vsi_iso_instance_count is set to 1, its not going to install the iso node.

Reply:
yeah that's a nice thing. Sure let me update the PR